src/Parser/DisciplinePriest/Modules/Spells/Atonement.js

```diff
@@ -8,6 +8,7 @@ const debug = false;
 const IMPROPER_REFRESH_TIME = 3000;
 
 class Atonement extends Module {
+  priority = 99;
```

Comment:
A priority of `9` would better fit this module. Do a find in files for `priority =` to find similar module priorities.